src/macros.js

```diff
@@ -127,19 +127,19 @@ defineMacro("\u211D", "\\mathbb{R}");
 defineMacro("\u2124", "\\mathbb{Z}");
 
 // Unicode Greek capital letters
-defineMacro("\u0391", "A");
-defineMacro("\u0392", "B");
-defineMacro("\u0395", "E");
-defineMacro("\u0396", "Z");
-defineMacro("\u0397", "H");
-defineMacro("\u0399", "I");
-defineMacro("\u039A", "K");
-defineMacro("\u039C", "M");
-defineMacro("\u039D", "N");
-defineMacro("\u039F", "O");
-defineMacro("\u03A1", "P");
-defineMacro("\u03A4", "T");
-defineMacro("\u03A7", "X");
+defineMacro("\u0391", "{\\rm{A}}");
+defineMacro("\u0392", "{\\rm{B}}");
+defineMacro("\u0395", "{\\rm{E}}");
+defineMacro("\u0396", "{\\rm{Z}}");
+defineMacro("\u0397", "{\\rm{H}}");
+defineMacro("\u0399", "{\\rm{I}}");
+defineMacro("\u039A", "{\\rm{K}}");
+defineMacro("\u039C", "{\\rm{M}}");
+defineMacro("\u039D", "{\\rm{N}}");
+defineMacro("\u039F", "{\\rm{O}}");
+defineMacro("\u03A1", "{\\rm{P}}");
+defineMacro("\u03A4", "{\\rm{T}}");
+defineMacro("\u03A7", "{\\rm{X}}");
```

Comment:
Sorry, could you also remove the braces around the letter? We just need `{\\rm A}` for example.